opentracing-api/src/main/java/io/opentracing/ThreadLocalScheduler.java

```diff
@@ -0,0 +1,70 @@
+package io.opentracing;
+
+/**
+ * ThreadLocalScheduler is a trivial Scheduler implementation that relies on Java's thread-local storage primitives.
+ *
+ * @see Scheduler
+ * @see Tracer#scheduler()
+ */
+public class ThreadLocalScheduler implements Scheduler {
```

Comment:
@pavolloffay placing it alongside GlobalTracer (wherever it eventually lands) seems reasonable, good suggestion